src/main/resources/application.properties

```diff
@@ -122,5 +122,5 @@ quarkus.keycloak.policy-enforcer.enable=true
 # This property is not effective when running the application in JVM or native modes
 quarkus.keycloak.devservices.realm-path=quarkus-realm.json
 
-
-
+# Select the vector store, defaults to ELASTICSEARCH
+vector.store.type=ELASTICSEARCH
```

Comment:
One minor ask, can you change this to 'vector.store.default.type' for consistency and so folks understand it is just a default

Reply:
Thanks for the suggestions. Renamed as per the suggestion and included other relevant changes to get ElasticSearch flow to work. Also, tested the code changes as shown below in an OpenShift deployment. 

![image](https://github.com/user-attachments/assets/3297aead-b0a3-4bea-9c33-db16ba64dc6b)
![image](https://github.com/user-attachments/assets/5257e372-ac61-4005-8150-4299cac5d5a1)